README.md

```diff
@@ -13,7 +13,7 @@ description: Demonstrates the usage of all XAML/WinUI controls in an interactive
 
 # XAML Controls Gallery
 
-![Xaml Gallery Screenshot](README_Images/XamlGalleryLanding.PNG)
+![Xaml Gallery Screenshot](https://i.imgur.com/W5dmkLl.png)

```

Comment:
I think it would be better to replace the existing image with the new image if we want to change it instead of relying on imgur for this image. You should be able to overwrite the file in the folder and commit the new file together with this change.

Reply:
+1. It should be possible to update the image in the GitHub repo. We don't want to rely on imgur links, especially because we use the Gallery in situations where an internet connection isn't always guaranteed.

Reply:
ah okay, done that, thanks!